src/material/datepicker/datepicker.md

```diff
@@ -210,20 +210,34 @@ It's also possible to set the locale at runtime using the `setLocale` method of
 The datepicker was built to be date implementation agnostic. This means that it can be made to work
 with a variety of different date implementations. However it also means that developers need to make
 sure to provide the appropriate pieces for the datepicker to work with their chosen implementation.
-The easiest way to ensure this is just to import one of the pre-made modules:
 
-|Module               |Date type|Supported locales                                                      |Dependencies                      |Import from                       |
-|---------------------|---------|-----------------------------------------------------------------------|----------------------------------|----------------------------------|
-|`MatNativeDateModule`|`Date`   |en-US                                                                  |None                              |`@angular/material`               |
-|`MatMomentDateModule`|`Moment` |[See project](https://github.com/moment/moment/tree/develop/src/locale)|[Moment.js](https://momentjs.com/)|`@angular/material-moment-adapter`|
+The easiest way to ensure this is to import one of the provided date modules:
 
-*Please note: `MatNativeDateModule` is based off of the functionality available in JavaScript's
-native `Date` object, and is thus not suitable for many locales. One of the biggest shortcomings of
-the native `Date` object is the inability to set the parse format. We highly recommend using the
-`MomentDateAdapter` or a custom `DateAdapter` that works with the formatting/parsing library of your
-choice.*
+`MatNativeDateModule`
 
-These modules include providers for `DateAdapter` and `MAT_DATE_FORMATS`
+|                     |                        |
```

Comment:
I'm not sure the empty header row would look okay in the docs site. We could instead embed the raw HTML that would be more appropriate for this format, e.g.
```html
<table>
  <tbody>
  <tr>
    <th align="left" scope="row">Date type</th>
    <td><code>Date</code></td>
  </tr>
  <tr>
    <th align="left" scope="row">Supported locales</th>
    <td>en-US</td>
  </tr>
  <tr>
    <th align="left" scope="row">Dependencies</th>
    <td>None</td>
  </tr>
  <tr>
    <th align="left" scope="row">Import from</th>
    <td><code>@angular/material/core</code></td>
  </tr>
  </tbody>
</table>
```